support-frontend/assets/pages/[countryGroupId]/thankYou.tsx

```diff
@@ -168,7 +164,7 @@ export function ThankYou({
 
 	if (
 		isSwitchOn('featureSwitches.enableThankYouOnboarding') &&
-		PRODUCTS_WITH_THANK_YOU_ONBOARDING.includes(productKey) &&
+    productKey !== undefined && appConfig.settings.productsWithThankYouOnboarding.includes(productKey) &&
```

Comment:
the type of the old `PRODUCTS_WITH_THANK_YOU_ONBOARDING`  included undefined which didn't seem to make sense in the new world, but I did have to add an undefined check to keep things happy.